### ISSUE TITLE
Difficult selecting the media start node for a user or a user group

### ISSUE THREAD

**Issue by engern** [umbraco/Umbraco-CMS#17785]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

15.1.1

### Bug summary

~~As an administrator I'm not able to set a media start node for auser or a user group~~
As an administrator I have a hard time figuring out how to pick a media folder as the media start node.

https://github.com/user-attachments/assets/577eb6e6-195e-4b42-8c7c-e803b52722ae

### Steps to reproduce

1. Navigate to Users -> User Groups
2. Edit a user group
3. Try to select a media start node

### Expected result / actual result

~~The selected folder should be set as the media start node for the user group~~
Picking a Start Node for Media should use a Tree Picker style Modal/UX, this will give the user a better overview of the structure. And as well solving the immediate problem of differentiating the click actions as the tree navigation has a expand-caret to unfold folders.

---
_This item has been added to our backlog AB#47432_

**Comment by nul800sebastiaan** [comment]
What if you click the title of the folder?

**Comment by engern** [comment]
@nul800sebastiaan Like this?  (I think the same applies to users too btw, not only user groups)

https://github.com/user-attachments/assets/0313e1eb-82e9-4f05-afff-5e1f215a8237

**Comment by nul800sebastiaan** [comment]
Since you can nest files under files I think they could still be a start nodes. 

Weird, I haven't tried to reproduce but I've seen at some point that selecting an icon doesn't work, but selecting the name does work. Guess that doesn't apply here!

**Comment by PeterKvayt** [comment]
Hi there!

I can't select any node if I click an icon or a name. Both options do not select the necessary node.

**Comment by VictorTavkin** [comment]
Hi!
I've faced with the same issue (v15.1.1).

**Comment by AndyButland** [comment]
I've looked into this one and believe it is working as expected.  The challenge here is that the click action needs to support two actions:

- Selecting the current item
- Opening the current item to reveal the sub-items and allow the selection or navigation from those

Due to this dual requirement the UX is difficult to make super clear, but the convention currently is that you select by clicking in the outer edge of the item, when it's highlighted with a blue border.  Clicking in the middle of the item will open it to show the sub items.

I find if I do click around the edge like this to select...

![image](https://github.com/user-attachments/assets/dc3cadae-645e-46b1-b543-d93e4062c456)

... and then click "Submit", the folder chosen is selected as you'd expect.

As such I'll close this issue.

**Comment by nielslyngsoe** [comment]
I do agree with Andy, but there is a user-experience problem in this case that I'm aware about.
If you have any suggestions to how you see this visually solved in a more intuitive way, please let me know.

On this specific case of "Picking a Start Node", I have created a task in our internal backlog regarding switching it to use a Tree based navigation. This will provide the user a better overview of the structure and as well solve the immediate problem of distinction between open(expand children) and selection(Picking the item).

This is thought not first priority(so feel free to provide a PR if you like), otherwise in the mean time I hope the learning from Andy helps.

**Comment by nul800sebastiaan** [comment]
If I remember correctly, the pattern used to be: click on the icon/preview image and that means you're selecting the item. Click on the item's name and that means you're going into the item.

This is how Dropbox does it, for example.

Of course the item doesn't have to be a "container" so when there is no underlying items, clicking on the item name either doesn't do anything or it just selects the item (but that last option would be inconsistent behavior!).

**Comment by nielslyngsoe** [comment]
@nul800sebastiaan actually the experience on Dropbox is if the item is not a container, clicking the thumbnail opens it.. So it is inconsistent. — Or in other words, they decided you most likely want to open items and select containers, one might say that is the opposite of what most users want... but depends on the mission 🤓

But they do have a checkbox top right, which you always can trust does a selection. This was also my idea on how we could go about it if it turns out to be a problem.

With above correction, the only place is this a problem is within the Media Picker configured to allow picking a Container-type(in general that is only folders)

**Comment by nielslyngsoe** [comment]
@nul800sebastiaan yes, that was how it was until 15.1. So very recent change. We saw a lot of people finding it unintuitive for navigation in the media library, generally all Card overviews.

That suggestion was specific for the Start Node Picking, where the actual thumbnails does not make much value, in that case the structure gives more value. :-)

**Comment by nielslyngsoe** [comment]
I'm reopening this issue, to use it as a reference for future UX correction.

**Comment by AndyButland** [comment]
Linking another report on this, which I'll close as a duplicate and redirect here: https://github.com/umbraco/Umbraco-CMS/issues/19533

**Comment by jemayn** [comment]
Found this issue while wanting to submit this as a bug as well 😅

Had a media picker where I wanted to set a start folder and I just could not get it to work. In the end I found if I tried highlighting the  card I could tell where the click and select borders were in order to find it.

![Image](https://github.com/user-attachments/assets/c3851df6-91bc-4b8c-87f3-cea8ed3cbaf8)

This feels really unintuitive! I'd even prefer the old way of having to know to click the name or the icon because atleast then once you knew it was clear where to click and not some invisible border area..

One more thing - if you do navigate into the folder instead of selecting it, and then press the big green "Choose" button, then it exits the modal but doesn't select it. I would assume that it would then select the folder I am currently looking into!

**Comment by nielslyngsoe** [comment]
For everyone who has been awaiting progress on this topic, then I can tell it is solved now by adding a checkbox top left corner on cards.
This is part of next release.

<img width="1004" height="406" alt="Image" src="https://github.com/user-attachments/assets/72db5eec-725e-4228-9b28-5d0d158857c1" />
